.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,4 +3,9 @@
 *Description of changes:*
 
 
-By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
+By submitting this pull request:
+*  I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
+* I confirm that no backwards incompatible change was made to Ace's public interface which is defined through the main typings file (`ace.d.ts`) and its references:
```

Comment:
Do you think we should mention what we consider a breaking change (modifications and deletions) or do you think that's obvious?